src/pages/CatIndex.test.js

```diff
@@ -0,0 +1,60 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom'
+import CatIndex from './CatIndex'
+import mockCats from '../mockCats'
+
+describe("<CatIndex />", () => {
+    it("renders a header", () => {
+        render(
```

Comment:
You can use a function to render the component and then call on that function before each assertion.

Reply:
Function used to render the component and is called before each assertion